site/src/pages/hub.page.tsx

```diff
@@ -27,14 +30,14 @@ export const getStaticProps: GetStaticProps<PageProps> = async () => {
 const HubPage: NextPage<PageProps> = ({ catalog }) => {
   return (
     <>
-      <Head>
-        <title>Block Protocol – Block Hub</title>
-        <meta itemProp="name" content="Block Hub" />
-        <meta
-          itemProp="description"
-          content="The Block Protocol's registry of open-source blocks"
-        />
-      </Head>
+      <NextSeo
+        title={seoTitle}
+        description={seoDesc}
+        openGraph={{
+          description: seoDesc,
+          title: seoTitle,
+        }}
```

Comment:
Do we need to define `openGraph` title and description if they match `title` and `description`? Current diff only sets `title` in docs, but this gets propagated to `og:title` in page source. Seems like `NextSeo` resolves these fields correctly.

I suggest either removing `openGraph` in this file or setting it elsewhere too.

Reply:
Tested the behavior of `NextSeo`, and you're right. only `title` / `description` is enough, no need to pass `openGraph` if we don't want to define sth else for openGraph.

Fixed here https://github.com/blockprotocol/blockprotocol/pull/562/commits/58f5c7ffb40d1cd0525e600d492933b31cb8bab8